.github_changelog_generator

```diff
@@ -1,4 +1,4 @@
-future-release=v1.12.1
-since-tag=v1.12.0
-date-format=%B %d, %Y
+future-release=v1.12.2
+since-tag=v1.12.1
+date-format=%Y %m, %d
```

Comment:
better use date-format=%Y-%m-%d to have like 2023-10-17.

Using %Y %m, %d will be like 2023 10, 17

Reply:
it makes sense. Updating

Reply:
updated